metagpt/provider/openai_api.py

```diff
@@ -323,3 +325,28 @@ async def amoderation(self, content: Union[str, list[str]]):
     async def _amoderation(self, content: Union[str, list[str]]):
         rsp = await self.llm.Moderation.acreate(input=content)
         return rsp
+
+    @staticmethod
+    def __get_rpm(config: Config) -> int:
+        session_key = config.get("OPENAI_SESSION_KEY", "")
+        default_rpm = int(config.get("RPM", 10))
+        if len(session_key) > 0:
+            try:
+                response = requests.get(
```

Comment:
Consider using `aiohttp` instead of `requests` for asynchronous and non-blocking HTTP requests.